01/21/2015 12:19:10|Trace|  Navigate to authentication page
01/21/2015 12:19:11|Trace|  Making log in
01/21/2015 12:19:14|Trace|  Getting dashboard JSON 
01/21/2015 12:19:14|Error|  Error on dashboard page
01/21/2015 12:19:14|Error| {"errorCode":0,"innerException":{"errorCode":0,"message":"ORA-12541: TNS:no listener","stackTrace":"   at Oracle.DataAccess.Client.OracleException.HandleErrorHelper(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, String procedure, Boolean bCheck, Int32 isRecoverable)\r\n   at Oracle.DataAccess.Client.OracleException.HandleError(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, Object src)\r\n   at Oracle.DataAccess.Client.OracleConnection.Open()\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.CreateBusinesTransaction()\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<CallApiVbxl>d__2c`1.MoveNext()","type":"Oracle.DataAccess.Client.OracleException, Oracle.DataAccess, Version=4.121.1.0, Culture=neutral, PublicKeyToken=89b483f429c47342","typeKey":"OracleException","typeName":"Oracle.DataAccess.Client.OracleException, Oracle.DataAccess, Version=4.121.1.0, Culture=neutral, PublicKeyToken=89b483f429c47342"},"message":"Ошибка базы данных Oracle. см. Внутреннее исключение","stackTrace":"   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<CallApiVbxl>d__2c`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<GetAgreementsAsync>d__70.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Helpers.CacheManager.<CachedCallAsync>d__5`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Helpers\\CacheManager.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Services.CacheServiceDataProvider.<GetFromRequestCache>d__0`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Services\\CacheService.cs:line 65\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Factory.ContractFactory.<>c__DisplayClass5.<<GetAgreementsMkAccount>b__0>d__a.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Factory\\ContractFactory.cs:line 46\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Helpers.CacheManager.<CachedCallAsync>d__5`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Helpers\\CacheManager.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Factory.ContractFactory.<GetContractsAsync>d__2c.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Factory\\ContractFactory.cs:line 183\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Factory.ContractFactory.<SearchContractsAsync>d__48.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Factory\\ContractFactory.cs:line 240\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Framework.Server.AsyncExtensions.<ProcessAsync>d__3`2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Server.ContractServices.<SearchAsync>d__d.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Dashboard.LetoDashboard.<GetDashboardResourceAsync>d__3.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Dashboard\\LetoDashboard.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Server.Core.SessionApiController.<Get>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Threading.Tasks.TaskHelpersExtensions.<CastToObject>d__3`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","type":"Ekassir.LetoBank.Integration.AbsAPI.AbsApiClientException, Ekassir.LetoBank.Integration.AbsAPI, Version=1.2.1.15, Culture=neutral, PublicKeyToken=null","typeKey":"AbsApiClientException","typeName":"Ekassir.LetoBank.Integration.AbsAPI.AbsApiClientException, Ekassir.LetoBank.Integration.AbsAPI, Version=1.2.1.15, Culture=neutral, PublicKeyToken=null"}
01/21/2015 12:19:14|Info|  Making screenshot ScreenShots\mtestSelf.jpg
01/21/2015 12:19:14|Trace|  Navigate to common timeline event page 
01/21/2015 12:19:17|Trace|  Getting common timeline event JSON
01/21/2015 12:19:17|Error|  Error on common timeline event page
01/21/2015 12:19:17|Error| {"errorCode":0,"innerException":{"errorCode":0,"message":"ORA-12541: TNS:no listener","stackTrace":"   at Oracle.DataAccess.Client.OracleException.HandleErrorHelper(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, String procedure, Boolean bCheck, Int32 isRecoverable)\r\n   at Oracle.DataAccess.Client.OracleException.HandleError(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, Object src)\r\n   at Oracle.DataAccess.Client.OracleConnection.Open()\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.CreateBusinesTransaction()\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<CallApiVbxl>d__2c`1.MoveNext()","type":"Oracle.DataAccess.Client.OracleException, Oracle.DataAccess, Version=4.121.1.0, Culture=neutral, PublicKeyToken=89b483f429c47342","typeKey":"OracleException","typeName":"Oracle.DataAccess.Client.OracleException, Oracle.DataAccess, Version=4.121.1.0, Culture=neutral, PublicKeyToken=89b483f429c47342"},"message":"Ошибка базы данных Oracle. см. Внутреннее исключение","stackTrace":"   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<CallApiVbxl>d__2c`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<GetAgreementsAsync>d__70.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Helpers.CacheManager.<CachedCallAsync>d__5`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Helpers\\CacheManager.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Services.CacheServiceDataProvider.<GetFromRequestCache>d__0`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Services\\CacheService.cs:line 65\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Factory.FactoryConnectorBase.<GetDateRangeOfContracts>d__b.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Factory\\FactoryConnectorBase.cs:line 37\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Factory.VaBankEventFactory.<HasHistoryEventsAsync>d__0.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Factory\\VaBankEventFactory.cs:line 72\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Connectors.LetoMainConnector.<HasHistoryEventsAsync>d__72.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Connectors\\LetoMainConnector.cs:line 578\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Framework.Server.AsyncExtensions.<ProcessAsync>d__3`2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Connectors.EventsHistory.EventsHistoryCacheConnector.<>c__DisplayClass8.<<GetHistoryEventsAsync>b__0>d__e.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Common.ICacheProviderExtensions.<ReturnFromCacheAsync>d__6`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Connectors.EventsHistory.EventsHistoryCacheConnector.<GetHistoryEventsAsync>d__11.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Server.Core.EventsHistoryService`2.<GetHistoryEventsAsync>d__37.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Server.Core.EventsHistorySegmentCreator`2.<CreateSegmentAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Server.Core.EventsHistorySegmentMap`2.<CreateRootSegmentAsync>d__1a.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Server.Core.EventsHistorySegmentMap`2.<GetRootSegmentAsync>d__1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Server.Core.EventsHistorySegmentMap`2.<FindSegmentForDateAsync>d__16.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Server.Core.EventsHistoryService`2.<GetHistoryEventAsync>d__28.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Server.Core.TimelineApiController.<Get>d__f.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Threading.Tasks.TaskHelpersExtensions.<CastToObject>d__3`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","type":"Ekassir.LetoBank.Integration.AbsAPI.AbsApiClientException, Ekassir.LetoBank.Integration.AbsAPI, Version=1.2.1.15, Culture=neutral, PublicKeyToken=null","typeKey":"AbsApiClientException","typeName":"Ekassir.LetoBank.Integration.AbsAPI.AbsApiClientException, Ekassir.LetoBank.Integration.AbsAPI, Version=1.2.1.15, Culture=neutral, PublicKeyToken=null"}
01/21/2015 12:19:17|Info|  Making screenshot ScreenShots\mtestTimelineEvent.jpg
01/21/2015 12:19:17|Trace|  Navigate to common timeline page 
01/21/2015 12:19:19|Trace|  Getting common timeline JSON
01/21/2015 12:19:19|Error|  Error on common timeline page
01/21/2015 12:19:19|Error| {"errorCode":0,"innerException":{"errorCode":0,"message":"ORA-12541: TNS:no listener","stackTrace":"   at Oracle.DataAccess.Client.OracleException.HandleErrorHelper(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, String procedure, Boolean bCheck, Int32 isRecoverable)\r\n   at Oracle.DataAccess.Client.OracleException.HandleError(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, Object src)\r\n   at Oracle.DataAccess.Client.OracleConnection.Open()\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.CreateBusinesTransaction()\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<CallApiVbxl>d__2c`1.MoveNext()","type":"Oracle.DataAccess.Client.OracleException, Oracle.DataAccess, Version=4.121.1.0, Culture=neutral, PublicKeyToken=89b483f429c47342","typeKey":"OracleException","typeName":"Oracle.DataAccess.Client.OracleException, Oracle.DataAccess, Version=4.121.1.0, Culture=neutral, PublicKeyToken=89b483f429c47342"},"message":"Ошибка базы данных Oracle. см. Внутреннее исключение","stackTrace":"   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<CallApiVbxl>d__2c`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<GetAgreementsAsync>d__70.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Helpers.CacheManager.<CachedCallAsync>d__5`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Helpers\\CacheManager.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Services.CacheServiceDataProvider.<GetFromRequestCache>d__0`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Services\\CacheService.cs:line 65\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Factory.FactoryConnectorBase.<GetDateRangeOfContracts>d__b.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Factory\\FactoryConnectorBase.cs:line 37\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Factory.VaBankEventFactory.<HasHistoryEventsAsync>d__0.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Factory\\VaBankEventFactory.cs:line 72\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Connectors.LetoMainConnector.<HasHistoryEventsAsync>d__72.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Connectors\\LetoMainConnector.cs:line 578\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Framework.Server.AsyncExtensions.<ProcessAsync>d__3`2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Connectors.EventsHistory.EventsHistoryCacheConnector.<>c__DisplayClass8.<<GetHistoryEventsAsync>b__0>d__e.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Common.ICacheProviderExtensions.<ReturnFromCacheAsync>d__6`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Connectors.EventsHistory.EventsHistoryCacheConnector.<GetHistoryEventsAsync>d__11.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Server.Core.EventsHistoryService`2.<GetHistoryEventsAsync>d__37.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Server.Core.EventsHistorySegmentCreator`2.<CreateSegmentAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Server.Core.EventsHistorySegmentMap`2.<CreateRootSegmentAsync>d__1a.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Server.Core.EventsHistorySegmentMap`2.<GetRootSegmentAsync>d__1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Server.Core.EventsHistoryService`2.<GetEventsHistoryPageAsync>d__6b.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Server.Core.EventsHistoryService`2.<GetEventsHistoryAsync>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Server.Core.TimelineApiController.<Get>d__7.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Threading.Tasks.TaskHelpersExtensions.<CastToObject>d__3`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","type":"Ekassir.LetoBank.Integration.AbsAPI.AbsApiClientException, Ekassir.LetoBank.Integration.AbsAPI, Version=1.2.1.15, Culture=neutral, PublicKeyToken=null","typeKey":"AbsApiClientException","typeName":"Ekassir.LetoBank.Integration.AbsAPI.AbsApiClientException, Ekassir.LetoBank.Integration.AbsAPI, Version=1.2.1.15, Culture=neutral, PublicKeyToken=null"}
01/21/2015 12:19:19|Info|  Making screenshot ScreenShots\mtestTimeline.jpg
01/21/2015 12:19:20|Trace|  Navigate to credit contract
01/21/2015 12:19:21|Trace|  Getting credit contract JSON: contractNumber 10020328
01/21/2015 12:19:21|Error|  Error on credit contract page
01/21/2015 12:19:21|Error| {"errorCode":0,"innerException":{"errorCode":0,"message":"ORA-12541: TNS:no listener","stackTrace":"   at Oracle.DataAccess.Client.OracleException.HandleErrorHelper(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, String procedure, Boolean bCheck, Int32 isRecoverable)\r\n   at Oracle.DataAccess.Client.OracleException.HandleError(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, Object src)\r\n   at Oracle.DataAccess.Client.OracleConnection.Open()\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.CreateBusinesTransaction()\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<CallApiVbxl>d__2c`1.MoveNext()","type":"Oracle.DataAccess.Client.OracleException, Oracle.DataAccess, Version=4.121.1.0, Culture=neutral, PublicKeyToken=89b483f429c47342","typeKey":"OracleException","typeName":"Oracle.DataAccess.Client.OracleException, Oracle.DataAccess, Version=4.121.1.0, Culture=neutral, PublicKeyToken=89b483f429c47342"},"message":"Ошибка базы данных Oracle. см. Внутреннее исключение","stackTrace":"   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<CallApiVbxl>d__2c`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<GetAgreementsAsync>d__70.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Helpers.CacheManager.<CachedCallAsync>d__5`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Helpers\\CacheManager.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Services.CacheServiceDataProvider.<GetFromRequestCache>d__0`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Services\\CacheService.cs:line 65\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Connectors.LetoMainConnector.<ValidateUserContract>d__ab.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Connectors\\LetoMainConnector.cs:line 697\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Connectors.LetoMainConnector.<ValidateUserContract>d__a3.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Connectors\\LetoMainConnector.cs:line 686\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Connectors.LetoMainConnector.<ValidateUserContract>d__9c.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Connectors\\LetoMainConnector.cs:line 673\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Connectors.LetoMainConnector.<GetContractAsync>d__55.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Connectors\\LetoMainConnector.cs:line 494\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Framework.Server.AsyncExtensions.<ProcessAsync>d__3`2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Server.ContractServices.<GetContractAsync>d__2b.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Banking.Server.ContractsApiController.<GetByContractId>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Threading.Tasks.TaskHelpersExtensions.<CastToObject>d__3`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","type":"Ekassir.LetoBank.Integration.AbsAPI.AbsApiClientException, Ekassir.LetoBank.Integration.AbsAPI, Version=1.2.1.15, Culture=neutral, PublicKeyToken=null","typeKey":"AbsApiClientException","typeName":"Ekassir.LetoBank.Integration.AbsAPI.AbsApiClientException, Ekassir.LetoBank.Integration.AbsAPI, Version=1.2.1.15, Culture=neutral, PublicKeyToken=null"}
01/21/2015 12:19:21|Info|  Making screenshot ScreenShots\mtestContract.jpg
01/21/2015 12:19:21|Trace|  Navigate to contract services 
01/21/2015 12:19:21|Error|  No JSON on contracts services page
01/21/2015 12:19:21|Info|  Making scrennshot
01/21/2015 12:19:22|Trace|  Getting JSON /banking/contracts/10020328/services
01/21/2015 12:19:22|Trace|  Navigate to all user's contracts page
01/21/2015 12:19:23|Trace|  Getting user's contracts JSON
01/21/2015 12:19:23|Error|  Error on user's contracts page
01/21/2015 12:19:23|Error| {"errorCode":0,"innerException":{"errorCode":0,"message":"ORA-12541: TNS:no listener","stackTrace":"   at Oracle.DataAccess.Client.OracleException.HandleErrorHelper(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, String procedure, Boolean bCheck, Int32 isRecoverable)\r\n   at Oracle.DataAccess.Client.OracleException.HandleError(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, Object src)\r\n   at Oracle.DataAccess.Client.OracleConnection.Open()\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.CreateBusinesTransaction()\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<CallApiVbxl>d__2c`1.MoveNext()","type":"Oracle.DataAccess.Client.OracleException, Oracle.DataAccess, Version=4.121.1.0, Culture=neutral, PublicKeyToken=89b483f429c47342","typeKey":"OracleException","typeName":"Oracle.DataAccess.Client.OracleException, Oracle.DataAccess, Version=4.121.1.0, Culture=neutral, PublicKeyToken=89b483f429c47342"},"message":"Ошибка базы данных Oracle. см. Внутреннее исключение","stackTrace":"   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<CallApiVbxl>d__2c`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<GetAgreementsAsync>d__70.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Helpers.CacheManager.<CachedCallAsync>d__5`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Helpers\\CacheManager.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Services.CacheServiceDataProvider.<GetFromRequestCache>d__0`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Services\\CacheService.cs:line 65\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Factory.ContractFactory.<>c__DisplayClass5.<<GetAgreementsMkAccount>b__0>d__a.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Factory\\ContractFactory.cs:line 46\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Helpers.CacheManager.<CachedCallAsync>d__5`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Helpers\\CacheManager.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Factory.ContractFactory.<GetContractsAsync>d__2c.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Factory\\ContractFactory.cs:line 183\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Factory.ContractFactory.<SearchContractsAsync>d__48.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Factory\\ContractFactory.cs:line 240\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Framework.Server.AsyncExtensions.<ProcessAsync>d__3`2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Server.ContractServices.<SearchAsync>d__d.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Banking.Server.ContractsApiController.<GetContracts>d__a.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Threading.Tasks.TaskHelpersExtensions.<CastToObject>d__3`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","type":"Ekassir.LetoBank.Integration.AbsAPI.AbsApiClientException, Ekassir.LetoBank.Integration.AbsAPI, Version=1.2.1.15, Culture=neutral, PublicKeyToken=null","typeKey":"AbsApiClientException","typeName":"Ekassir.LetoBank.Integration.AbsAPI.AbsApiClientException, Ekassir.LetoBank.Integration.AbsAPI, Version=1.2.1.15, Culture=neutral, PublicKeyToken=null"}
01/21/2015 12:19:23|Info|  Making screenshot ScreenShots\mtestContracts.jpg
01/21/2015 12:19:23|Trace|  Navigate to contract timeline event page 
01/21/2015 12:19:25|Trace|  Getting contract timeline event JSON
01/21/2015 12:19:25|Error|  Error on contract timeline event page
01/21/2015 12:19:25|Error| {"errorCode":0,"innerException":{"errorCode":0,"message":"ORA-12541: TNS:no listener","stackTrace":"   at Oracle.DataAccess.Client.OracleException.HandleErrorHelper(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, String procedure, Boolean bCheck, Int32 isRecoverable)\r\n   at Oracle.DataAccess.Client.OracleException.HandleError(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, Object src)\r\n   at Oracle.DataAccess.Client.OracleConnection.Open()\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.CreateBusinesTransaction()\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<CallApiVbxl>d__2c`1.MoveNext()","type":"Oracle.DataAccess.Client.OracleException, Oracle.DataAccess, Version=4.121.1.0, Culture=neutral, PublicKeyToken=89b483f429c47342","typeKey":"OracleException","typeName":"Oracle.DataAccess.Client.OracleException, Oracle.DataAccess, Version=4.121.1.0, Culture=neutral, PublicKeyToken=89b483f429c47342"},"message":"Ошибка базы данных Oracle. см. Внутреннее исключение","stackTrace":"   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<CallApiVbxl>d__2c`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<GetAgreementsAsync>d__70.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Helpers.CacheManager.<CachedCallAsync>d__5`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Helpers\\CacheManager.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Services.CacheServiceDataProvider.<GetFromRequestCache>d__0`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Services\\CacheService.cs:line 65\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Connectors.LetoMainConnector.<ValidateUserContract>d__ab.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Connectors\\LetoMainConnector.cs:line 697\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Connectors.LetoMainConnector.<ValidateUserContract>d__a3.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Connectors\\LetoMainConnector.cs:line 686\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Connectors.LetoMainConnector.<ValidateUserContract>d__9c.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Connectors\\LetoMainConnector.cs:line 673\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Connectors.LetoMainConnector.<GetContractAsync>d__55.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Connectors\\LetoMainConnector.cs:line 494\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Framework.Server.AsyncExtensions.<ProcessAsync>d__3`2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Server.ContractServices.<GetContractAsync>d__2b.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Server.ContractTimelineApiController.<GetByEventId>d__11.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Threading.Tasks.TaskHelpersExtensions.<CastToObject>d__3`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","type":"Ekassir.LetoBank.Integration.AbsAPI.AbsApiClientException, Ekassir.LetoBank.Integration.AbsAPI, Version=1.2.1.15, Culture=neutral, PublicKeyToken=null","typeKey":"AbsApiClientException","typeName":"Ekassir.LetoBank.Integration.AbsAPI.AbsApiClientException, Ekassir.LetoBank.Integration.AbsAPI, Version=1.2.1.15, Culture=neutral, PublicKeyToken=null"}
01/21/2015 12:19:25|Info|  Making screenshot ScreenShots\mtestContractTimelineEvent.jpg
01/21/2015 12:19:25|Trace|  Navigate to contract timeline page 
01/21/2015 12:19:26|Trace|  Getting JSON /banking/contracts/10020328/timeline
01/21/2015 12:19:26|Error|  Error on contract timeline page
01/21/2015 12:19:26|Error| {"errorCode":0,"innerException":{"errorCode":0,"message":"ORA-12541: TNS:no listener","stackTrace":"   at Oracle.DataAccess.Client.OracleException.HandleErrorHelper(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, String procedure, Boolean bCheck, Int32 isRecoverable)\r\n   at Oracle.DataAccess.Client.OracleException.HandleError(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, Object src)\r\n   at Oracle.DataAccess.Client.OracleConnection.Open()\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.CreateBusinesTransaction()\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<CallApiVbxl>d__2c`1.MoveNext()","type":"Oracle.DataAccess.Client.OracleException, Oracle.DataAccess, Version=4.121.1.0, Culture=neutral, PublicKeyToken=89b483f429c47342","typeKey":"OracleException","typeName":"Oracle.DataAccess.Client.OracleException, Oracle.DataAccess, Version=4.121.1.0, Culture=neutral, PublicKeyToken=89b483f429c47342"},"message":"Ошибка базы данных Oracle. см. Внутреннее исключение","stackTrace":"   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<CallApiVbxl>d__2c`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.LetoBank.Integration.AbsAPI.ApiVbxlFunctions.AbsApiClient.<GetAgreementsAsync>d__70.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Helpers.CacheManager.<CachedCallAsync>d__5`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Helpers\\CacheManager.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Services.CacheServiceDataProvider.<GetFromRequestCache>d__0`1.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Services\\CacheService.cs:line 65\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Connectors.LetoMainConnector.<ValidateUserContract>d__ab.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Connectors\\LetoMainConnector.cs:line 697\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Connectors.LetoMainConnector.<ValidateUserContract>d__a3.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Connectors\\LetoMainConnector.cs:line 686\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Connectors.LetoMainConnector.<ValidateUserContract>d__9c.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Connectors\\LetoMainConnector.cs:line 673\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Connectors.Leto.Connectors.LetoMainConnector.<GetContractAsync>d__55.MoveNext() in d:\\Projects\\LetoBank\\PersonalCabinet\\Ekassir.PersonalCabinet.Banking.Connectors.Leto\\Connectors\\LetoMainConnector.cs:line 494\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Framework.Server.AsyncExtensions.<ProcessAsync>d__3`2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Ekassir.PersonalCabinet.Banking.Server.ContractServices.<GetContractAsync>d__2b.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Ekassir.PersonalCabinet.Banking.Server.ContractTimelineApiController.<Get>d__7.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Threading.Tasks.TaskHelpersExtensions.<CastToObject>d__3`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","type":"Ekassir.LetoBank.Integration.AbsAPI.AbsApiClientException, Ekassir.LetoBank.Integration.AbsAPI, Version=1.2.1.15, Culture=neutral, PublicKeyToken=null","typeKey":"AbsApiClientException","typeName":"Ekassir.LetoBank.Integration.AbsAPI.AbsApiClientException, Ekassir.LetoBank.Integration.AbsAPI, Version=1.2.1.15, Culture=neutral, PublicKeyToken=null"}
01/21/2015 12:19:26|Info|  Making screenshot ScreenShots\mtestContractTimeline.jpg
01/21/2015 12:19:27|Trace|  Navigate to credit's payments schedule
01/21/2015 12:19:28|Trace|  Getting credit's payments schedule JSON
01/21/2015 12:19:28|Error|  Error on credit's payments schedule page
01/21/2015 12:19:28|Error| {"message":"An error has occurred.","exceptionMessage":"Object reference not set to an instance of an object.","exceptionType":"System.NullReferenceException","stackTrace":"   at Ekassir.PersonalCabinet.Common.Logging.LoggingExtensions.FilteredLog(ILogger logger, LogLevel level, Exception exception, String format, Object[] objects)\r\n   at Ekassir.PersonalCabinet.Common.Logging.LoggingExtensions.Error(ILogger logger, Exception exception, String message)\r\n   at Ekassir.PersonalCabinet.Framework.Server.CabinetApiExceptionFilterAttribute.OnException(HttpActionExecutedContext actionExecutedContext)\r\n   at System.Web.Http.Filters.ExceptionFilterAttribute.OnExceptionAsync(HttpActionExecutedContext actionExecutedContext, CancellationToken cancellationToken)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.ExceptionFilterAttribute.<ExecuteExceptionFilterAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at System.Web.Http.Dispatcher.HttpControllerDispatcher.<SendAsync>d__1.MoveNext()"}
01/21/2015 12:19:28|Info|  Making screenshot ScreenShots\mtestContractSchedule.jpg
01/21/2015 14:12:10|Trace|  Navigate to authentication page
01/21/2015 14:12:11|Trace|  Filling authentication fields
01/21/2015 14:12:12|Trace|  Click Registration Button 
01/21/2015 14:12:12|Trace|  Agree with license terms
01/21/2015 14:12:12|Trace|  Choose registration with account
01/21/2015 14:12:13|Trace|  Enter Account Number and access code: 40817810200340165498 123456
01/21/2015 14:12:13|Trace|  Click Next. Go to SMS code page
01/21/2015 14:12:20|Error|  Выполнение операции невозможно по техническим причинам
01/21/2015 14:12:20|Trace|  Enter SMS code(otp)
01/21/2015 14:12:26|Trace|  Navigate to authentication page
01/21/2015 14:12:27|Trace|  Filling authentication fields
01/21/2015 14:12:27|Trace|  Click Registration Button 
01/21/2015 14:12:28|Trace|  Agree with license terms
01/21/2015 14:12:28|Trace|  Choose registration with account
01/21/2015 14:12:28|Trace|  Enter Account Number and access code: 40817810800340005165 123456
01/21/2015 14:12:29|Trace|  Click Next. Go to SMS code page
01/21/2015 14:12:49|Trace|  Enter SMS code(otp)
01/21/2015 14:12:49|Trace|  Click Next. Go to username page
01/21/2015 14:12:50|Trace|  Enter username: u613385
01/21/2015 14:12:50|Trace|  Delete existing username field
01/21/2015 14:12:50|Trace|  Click Next. Go to password page
01/21/2015 14:13:22|Trace|  Enter and confirm password
01/21/2015 14:13:23|Trace|  Click Next. Go to protect code page
01/21/2015 14:13:23|Trace|  Fill protect code fields
01/21/2015 14:13:23|Trace|  Click dismiss setting protect code
01/21/2015 14:13:46|Trace|  Navigate to authentication page
01/21/2015 14:13:47|Trace|  Filling authentication fields
01/21/2015 14:13:47|Trace|  Click Registration Button 
01/21/2015 14:13:48|Trace|  Agree with license terms
01/21/2015 14:13:48|Trace|  Choose registration with account
01/21/2015 14:13:48|Trace|  Enter Account Number and access code: 40817810400340046087 123456
01/21/2015 14:13:48|Trace|  Click Next. Go to SMS code page
01/21/2015 14:14:06|Trace|  Enter SMS code(otp)
01/21/2015 14:14:06|Trace|  Click Next. Go to username page
01/21/2015 14:14:07|Trace|  Enter username: u5992451
01/21/2015 14:14:07|Trace|  Delete existing username field
01/21/2015 14:14:07|Trace|  Click Next. Go to password page
01/21/2015 14:14:29|Trace|  Enter and confirm password
01/21/2015 14:14:29|Trace|  Click Next. Go to protect code page
01/21/2015 14:14:30|Trace|  Fill protect code fields
01/21/2015 14:14:30|Trace|  Click dismiss setting protect code
01/21/2015 14:14:43|Trace|  Navigate to authentication page
01/21/2015 14:14:44|Trace|  Filling authentication fields
01/21/2015 14:14:44|Trace|  Click Registration Button 
01/21/2015 14:14:45|Trace|  Agree with license terms
01/21/2015 14:14:45|Trace|  Choose registration with account
01/21/2015 14:14:45|Trace|  Enter Account Number and access code: 40817810100340062426 123456
01/21/2015 14:14:45|Trace|  Click Next. Go to SMS code page
01/21/2015 14:15:01|Trace|  Enter SMS code(otp)
01/21/2015 14:15:01|Trace|  Click Next. Go to username page
01/21/2015 14:15:02|Trace|  Enter username: u6064915
01/21/2015 14:15:02|Trace|  Delete existing username field
01/21/2015 14:15:02|Trace|  Click Next. Go to password page
01/21/2015 14:15:20|Trace|  Enter and confirm password
01/21/2015 14:15:20|Trace|  Click Next. Go to protect code page
01/21/2015 14:15:21|Trace|  Fill protect code fields
01/21/2015 14:15:21|Trace|  Click dismiss setting protect code
01/21/2015 14:15:36|Trace|  Navigate to authentication page
01/21/2015 14:15:37|Trace|  Filling authentication fields
01/21/2015 14:15:37|Trace|  Click Registration Button 
01/21/2015 14:15:38|Trace|  Agree with license terms
01/21/2015 14:15:38|Trace|  Choose registration with account
01/21/2015 14:15:38|Trace|  Enter Account Number and access code: 40817810200340104666 123456
01/21/2015 14:15:38|Trace|  Click Next. Go to SMS code page
01/21/2015 14:15:56|Trace|  Enter SMS code(otp)
01/21/2015 14:15:56|Trace|  Click Next. Go to username page
01/21/2015 14:15:57|Trace|  Enter username: u5356338
01/21/2015 14:15:57|Trace|  Delete existing username field
01/21/2015 14:15:57|Trace|  Click Next. Go to password page
01/21/2015 14:16:17|Trace|  Enter and confirm password
01/21/2015 14:16:17|Trace|  Click Next. Go to protect code page
01/21/2015 14:16:17|Trace|  Fill protect code fields
01/21/2015 14:16:18|Trace|  Click dismiss setting protect code
01/21/2015 14:16:29|Trace|  Navigate to authentication page
01/21/2015 14:16:31|Trace|  Filling authentication fields
01/21/2015 14:16:31|Trace|  Click Registration Button 
01/21/2015 14:16:32|Trace|  Agree with license terms
01/21/2015 14:16:32|Trace|  Choose registration with account
01/21/2015 14:16:32|Trace|  Enter Account Number and access code: 40817810200340091711 123456
01/21/2015 14:16:33|Trace|  Click Next. Go to SMS code page
01/21/2015 14:16:50|Trace|  Enter SMS code(otp)
01/21/2015 14:16:50|Trace|  Click Next. Go to username page
01/21/2015 14:16:50|Trace|  Enter username: u137479
01/21/2015 14:16:50|Trace|  Delete existing username field
01/21/2015 14:16:50|Trace|  Click Next. Go to password page
01/21/2015 14:17:08|Trace|  Enter and confirm password
01/21/2015 14:17:09|Trace|  Click Next. Go to protect code page
01/21/2015 14:17:09|Trace|  Fill protect code fields
01/21/2015 14:17:09|Trace|  Click dismiss setting protect code
01/21/2015 14:17:25|Trace|  Navigate to authentication page
01/21/2015 14:17:25|Trace|  Filling authentication fields
01/21/2015 14:17:26|Trace|  Click Registration Button 
01/21/2015 14:17:26|Trace|  Agree with license terms
01/21/2015 14:17:26|Trace|  Choose registration with account
01/21/2015 14:17:27|Trace|  Enter Account Number and access code: 40817810400340001310 123456
01/21/2015 14:17:27|Trace|  Click Next. Go to SMS code page
01/21/2015 14:17:47|Trace|  Enter SMS code(otp)
01/21/2015 14:17:47|Trace|  Click Next. Go to username page
01/21/2015 14:17:48|Trace|  Enter username: u272972
01/21/2015 14:17:48|Trace|  Delete existing username field
01/21/2015 14:17:48|Trace|  Click Next. Go to password page
01/21/2015 14:18:08|Trace|  Enter and confirm password
01/21/2015 14:18:08|Trace|  Click Next. Go to protect code page
01/21/2015 14:18:09|Trace|  Fill protect code fields
01/21/2015 14:18:09|Trace|  Click dismiss setting protect code
01/21/2015 18:44:41|Trace|  Navigate to authentication page
01/21/2015 18:44:41|Trace|  Filling authentication fields
01/21/2015 18:44:42|Trace|  Click Registration Button 
01/21/2015 18:44:42|Trace|  Agree with license terms
01/21/2015 18:44:43|Trace|  Choose registration with account
01/21/2015 18:44:43|Trace|  Enter Account Number and access code: 40817810100340134642 123456
01/21/2015 18:44:43|Trace|  Click Next. Go to SMS code page
01/21/2015 18:45:10|Trace|  Enter SMS code(otp)
01/21/2015 18:45:10|Trace|  Click Next. Go to username page
01/21/2015 18:45:11|Trace|  Enter username: u156738
01/21/2015 18:45:11|Trace|  Delete existing username field
01/21/2015 18:45:11|Trace|  Click Next. Go to password page
01/21/2015 18:45:11|Trace|  Enter and confirm password
01/21/2015 18:45:12|Trace|  Click Next. Go to protect code page
01/21/2015 18:45:12|Trace|  Fill protect code fields
01/21/2015 18:45:13|Trace|  Click dismiss setting protect code
01/21/2015 19:08:41|Trace|  Navigate to authentication page
01/21/2015 19:08:46|Trace|  Making log in
01/21/2015 19:08:58|Trace|  Getting dashboard JSON 
01/21/2015 19:08:58|Info|  OK
01/21/2015 19:08:58|Trace|  Navigate to common timeline event page 
01/21/2015 19:09:05|Trace|  Getting common timeline event JSON
01/21/2015 19:09:05|Info|  OK
01/21/2015 19:09:05|Trace|  Navigate to common timeline page 
01/21/2015 19:09:15|Trace|  Getting common timeline JSON
01/21/2015 19:09:15|Info|  OK
01/21/2015 19:09:15|Trace|  Navigate to credit contract
01/21/2015 19:09:24|Trace|  Getting credit contract JSON: contractNumber 10020328
01/21/2015 19:09:24|Info|  OK
01/21/2015 19:09:24|Trace|  Navigate to contract services 
01/21/2015 19:09:25|Error|  No JSON on contracts services page
01/21/2015 19:09:25|Info|  Making scrennshot:
01/21/2015 19:09:25|Trace|  Getting JSON /banking/contracts/10020328/services
01/21/2015 19:09:25|Trace|  Navigate to all user's contracts page
01/21/2015 19:09:26|Trace|  Getting user's contracts JSON
01/21/2015 19:09:26|Info|  OK
01/21/2015 19:09:26|Trace|  Navigate to contract timeline event page 
01/21/2015 19:09:34|Trace|  Getting contract timeline event JSON
01/21/2015 19:09:34|Info|  OK
01/21/2015 19:09:34|Trace|  Navigate to contract timeline page 
01/21/2015 19:10:09|Trace|  Getting JSON /banking/contracts/10020328/timeline
01/21/2015 19:10:09|Info|  OK
01/21/2015 19:10:09|Trace|  Navigate to credit's payments schedule
01/21/2015 19:10:09|Trace|  Getting credit's payments schedule JSON
01/21/2015 19:10:09|Info|  OK
01/21/2015 19:11:42|Trace|  Navigate to authentication page
01/21/2015 19:11:53|Trace|  Filling authentication fields
01/21/2015 19:11:53|Trace|  Click Registration Button 
01/21/2015 19:11:53|Trace|  Agree with license terms
01/21/2015 19:11:55|Trace|  Choose registration with account
01/21/2015 19:11:57|Trace|  Enter Account Number and access code: 40817810600140528961 123456
01/21/2015 19:11:58|Trace|  Click Next. Go to SMS code page
01/21/2015 19:12:24|Trace|  Enter SMS code(otp)
01/21/2015 19:12:24|Trace|  Click Next. Go to username page
01/21/2015 19:12:25|Trace|  Enter username: u6502088
01/21/2015 19:12:25|Trace|  Delete existing username field
01/21/2015 19:12:25|Trace|  Click Next. Go to password page
01/21/2015 19:12:58|Trace|  Enter and confirm password
01/21/2015 19:12:59|Trace|  Click Next. Go to protect code page
01/21/2015 19:12:59|Trace|  Fill protect code fields
01/21/2015 19:12:59|Trace|  Click dismiss setting protect code
01/22/2015 12:39:15|Trace|  Navigate to authentication page
01/22/2015 12:39:17|Trace|  Filling authentication fields
01/22/2015 12:39:17|Trace|  Click Registration Button 
01/22/2015 12:39:18|Trace|  Agree with license terms
01/22/2015 12:39:18|Trace|  Choose registration with account
01/22/2015 12:39:19|Trace|  Enter Account Number and access code: 40817810000340008001 123456
01/22/2015 12:39:19|Trace|  Click Next. Go to SMS code page
01/22/2015 12:39:42|Trace|  Enter SMS code(otp)
01/22/2015 12:39:42|Trace|  Click Next. Go to username page
01/22/2015 12:39:43|Trace|  Enter username: u55956
01/22/2015 12:39:43|Trace|  Delete existing username field
01/22/2015 12:39:43|Trace|  Click Next. Go to password page
01/22/2015 12:39:43|Trace|  Enter and confirm password
01/22/2015 12:39:44|Trace|  Click Next. Go to protect code page
01/22/2015 12:39:44|Trace|  Fill protect code fields
01/22/2015 12:39:44|Trace|  Click dismiss setting protect code
01/22/2015 12:40:05|Trace|  Navigate to authentication page
01/22/2015 12:40:06|Trace|  Filling authentication fields
01/22/2015 12:40:07|Trace|  Click Registration Button 
01/22/2015 12:40:07|Trace|  Agree with license terms
01/22/2015 12:40:07|Trace|  Choose registration with account
01/22/2015 12:40:07|Trace|  Enter Account Number and access code: 40817810100340001241 123456
01/22/2015 12:40:08|Trace|  Click Next. Go to SMS code page
01/22/2015 12:40:27|Trace|  Enter SMS code(otp)
01/22/2015 12:40:27|Trace|  Click Next. Go to username page
01/22/2015 12:40:28|Trace|  Enter username: u5596623
01/22/2015 12:40:28|Trace|  Delete existing username field
01/22/2015 12:40:28|Trace|  Click Next. Go to password page
01/22/2015 12:40:53|Trace|  Enter and confirm password
01/22/2015 12:40:53|Trace|  Click Next. Go to protect code page
01/22/2015 12:40:54|Trace|  Fill protect code fields
01/22/2015 12:40:54|Trace|  Click dismiss setting protect code
01/22/2015 12:46:20|Trace|  Navigate to authentication page
01/22/2015 12:46:33|Trace|  Filling authentication fields
01/22/2015 12:46:33|Trace|  Click Registration Button 
01/22/2015 12:46:34|Trace|  Agree with license terms
01/22/2015 12:46:36|Trace|  Choose registration with account
01/22/2015 12:46:37|Trace|  Enter Account Number and access code: 40817810900340000798 123456
01/22/2015 12:46:38|Trace|  Click Next. Go to SMS code page
01/22/2015 12:46:50|Error|  Выполнение операции невозможно по техническим причинам
01/22/2015 12:46:50|Trace|  Enter SMS code(otp)
01/22/2015 12:46:55|Trace|  Navigate to authentication page
01/22/2015 12:47:16|Trace|  Filling authentication fields
01/22/2015 12:47:21|Trace|  Navigate to authentication page
01/22/2015 12:47:25|Trace|  Filling authentication fields
01/22/2015 12:47:25|Trace|  Click Registration Button 
01/22/2015 12:47:26|Trace|  Agree with license terms
01/22/2015 12:47:27|Trace|  Choose registration with account
01/22/2015 12:47:28|Trace|  Enter Account Number and access code: 40817810200340020850 123456
01/22/2015 12:47:34|Trace|  Navigate to authentication page
01/22/2015 12:47:47|Trace|  Filling authentication fields
01/22/2015 12:47:47|Trace|  Click Registration Button 
01/22/2015 12:47:48|Trace|  Agree with license terms
01/22/2015 12:47:48|Trace|  Choose registration with account
01/22/2015 12:47:49|Trace|  Enter Account Number and access code: 40817810900340031929 123456
01/22/2015 12:47:50|Trace|  Click Next. Go to SMS code page
01/22/2015 12:47:58|Error|  Выполнение операции невозможно по техническим причинам
01/22/2015 12:47:58|Trace|  Enter SMS code(otp)
01/22/2015 12:48:03|Trace|  Navigate to authentication page
01/22/2015 12:48:13|Trace|  Filling authentication fields
01/22/2015 12:48:13|Trace|  Click Registration Button 
01/22/2015 12:48:14|Trace|  Agree with license terms
01/22/2015 12:51:05|Trace|  Navigate to authentication page
01/22/2015 12:51:18|Trace|  Filling authentication fields
01/22/2015 12:51:18|Trace|  Click Registration Button 
01/22/2015 12:51:19|Trace|  Agree with license terms
01/22/2015 12:51:21|Trace|  Choose registration with account
01/22/2015 12:51:21|Trace|  Enter Account Number and access code: 40817810900340000798 123456
01/22/2015 12:51:22|Trace|  Click Next. Go to SMS code page
01/22/2015 12:51:48|Trace|  Enter SMS code(otp)
01/22/2015 12:51:48|Trace|  Click Next. Go to username page
01/22/2015 12:51:49|Trace|  Enter username: u5589895
01/22/2015 12:51:49|Trace|  Delete existing username field
01/22/2015 12:51:49|Trace|  Click Next. Go to password page
01/22/2015 12:51:51|Trace|  Enter and confirm password
01/22/2015 12:51:51|Trace|  Click Next. Go to protect code page
01/22/2015 12:51:55|Trace|  Fill protect code fields
01/22/2015 12:51:55|Trace|  Click dismiss setting protect code
01/22/2015 12:53:01|Trace|  Navigate to authentication page
01/22/2015 12:53:19|Trace|  Filling authentication fields
01/22/2015 12:53:19|Trace|  Click Registration Button 
01/22/2015 12:53:20|Trace|  Agree with license terms
01/22/2015 12:53:22|Trace|  Choose registration with account
01/22/2015 12:53:22|Trace|  Enter Account Number and access code: 40817810000340000423 123456
01/22/2015 12:53:23|Trace|  Click Next. Go to SMS code page
01/22/2015 12:53:31|Error|  Выполнение операции невозможно по техническим причинам
01/22/2015 12:53:31|Trace|  Enter SMS code(otp)
01/22/2015 12:53:36|Trace|  Navigate to authentication page
01/22/2015 12:53:37|Trace|  Filling authentication fields
01/22/2015 12:53:37|Trace|  Click Registration Button 
01/22/2015 12:53:37|Trace|  Agree with license terms
01/22/2015 12:53:37|Trace|  Choose registration with account
01/22/2015 12:53:38|Trace|  Enter Account Number and access code: 40817810200340020850 123456
01/22/2015 12:53:38|Trace|  Click Next. Go to SMS code page
01/22/2015 12:53:55|Trace|  Enter SMS code(otp)
01/22/2015 12:53:55|Trace|  Click Next. Go to username page
01/22/2015 12:53:55|Trace|  Enter username: u5799519
01/22/2015 12:53:55|Trace|  Delete existing username field
01/22/2015 12:53:55|Trace|  Click Next. Go to password page
01/22/2015 12:53:56|Trace|  Enter and confirm password
01/22/2015 12:53:56|Trace|  Click Next. Go to protect code page
01/22/2015 12:53:57|Trace|  Fill protect code fields
01/22/2015 12:53:57|Trace|  Click dismiss setting protect code
01/22/2015 12:54:09|Trace|  Navigate to authentication page
01/22/2015 12:54:18|Trace|  Filling authentication fields
01/22/2015 12:54:19|Trace|  Click Registration Button 
01/22/2015 12:54:19|Trace|  Agree with license terms
01/22/2015 12:54:21|Trace|  Choose registration with account
01/22/2015 12:54:25|Trace|  Enter Account Number and access code: 40817810900340031929 123456
01/22/2015 12:54:31|Trace|  Click Next. Go to SMS code page
01/22/2015 12:54:39|Error|  Выполнение операции невозможно по техническим причинам
01/22/2015 12:54:39|Trace|  Enter SMS code(otp)
01/22/2015 12:54:44|Trace|  Navigate to authentication page
01/22/2015 12:54:56|Trace|  Filling authentication fields
01/22/2015 12:54:57|Trace|  Click Registration Button 
01/22/2015 12:54:57|Trace|  Agree with license terms
01/22/2015 12:54:59|Trace|  Choose registration with account
01/22/2015 12:55:02|Trace|  Enter Account Number and access code: 40817810500340048228 123456
01/22/2015 12:55:04|Trace|  Click Next. Go to SMS code page
01/22/2015 12:55:24|Trace|  Enter SMS code(otp)
01/22/2015 12:55:24|Trace|  Click Next. Go to username page
01/22/2015 12:55:26|Trace|  Enter username: u6006507
01/22/2015 12:55:26|Trace|  Delete existing username field
01/22/2015 12:55:26|Trace|  Click Next. Go to password page
01/22/2015 12:55:53|Trace|  Enter and confirm password
01/22/2015 12:55:53|Trace|  Click Next. Go to protect code page
01/22/2015 12:55:56|Trace|  Fill protect code fields
01/22/2015 12:55:56|Trace|  Click dismiss setting protect code
01/22/2015 12:56:16|Trace|  Navigate to authentication page
01/22/2015 12:56:16|Trace|  Filling authentication fields
01/22/2015 12:56:17|Trace|  Click Registration Button 
01/22/2015 12:56:17|Trace|  Agree with license terms
01/22/2015 12:56:18|Trace|  Choose registration with account
01/22/2015 12:56:18|Trace|  Enter Account Number and access code: 40817810400340083370 123456
01/22/2015 12:56:18|Trace|  Click Next. Go to SMS code page
01/22/2015 12:56:36|Trace|  Enter SMS code(otp)
01/22/2015 12:56:36|Trace|  Click Next. Go to username page
01/22/2015 12:56:37|Trace|  Enter username: u6209928
01/22/2015 12:56:37|Trace|  Delete existing username field
01/22/2015 12:56:37|Trace|  Click Next. Go to password page
01/22/2015 12:57:00|Trace|  Enter and confirm password
01/22/2015 12:57:02|Trace|  Click Next. Go to protect code page
01/22/2015 12:57:02|Trace|  Fill protect code fields
01/22/2015 12:57:02|Trace|  Click dismiss setting protect code
01/22/2015 12:57:51|Trace|  Navigate to authentication page
01/22/2015 12:58:03|Trace|  Filling authentication fields
01/22/2015 12:58:04|Trace|  Click Registration Button 
01/22/2015 12:58:04|Trace|  Agree with license terms
01/22/2015 12:58:06|Trace|  Choose registration with account
01/22/2015 12:58:10|Trace|  Enter Account Number and access code: 40817810900340000798 123456
01/22/2015 12:58:12|Trace|  Click Next. Go to SMS code page
01/22/2015 12:58:34|Trace|  Enter SMS code(otp)
01/22/2015 12:58:34|Trace|  Click Next. Go to username page
01/22/2015 12:58:36|Trace|  Enter username: u5589895
01/22/2015 12:58:36|Trace|  Delete existing username field
01/22/2015 12:58:36|Trace|  Click Next. Go to password page
01/22/2015 12:58:37|Trace|  Enter and confirm password
01/22/2015 12:58:38|Trace|  Click Next. Go to protect code page
01/22/2015 12:58:40|Trace|  Fill protect code fields
01/22/2015 12:58:40|Trace|  Click dismiss setting protect code
01/22/2015 18:24:41|Trace|  Starting Web Driver
01/22/2015 18:24:42|Trace|  Making log in
01/22/2015 18:24:43|Trace|  Navigate to all user's contracts page
01/22/2015 18:24:43|Error|  No JSON on user's contracts page
01/22/2015 18:24:43|Info|  Making screenshot: ScreenShots\Contracts.jpg
01/22/2015 18:24:44|Trace|  Getting user's contracts JSON
01/22/2015 18:24:44|Trace|  Navigate to contract timeline event page 
01/22/2015 18:24:44|Error|  No JSON on contract timeline event page
01/22/2015 18:24:44|Info|  Making screenshot: ScreenShots\ContractTimelineEvent.jpg
01/22/2015 18:24:44|Trace|  Getting contract timeline event JSON
01/22/2015 18:24:44|Trace|  Navigate to contract timeline page 
01/22/2015 18:24:44|Error|  No JSON on contract timeline page
01/22/2015 18:24:44|Info|  Making screenshot: ScreenShots\ContractTimeline.jpg
01/22/2015 18:24:44|Trace|  Getting JSON /banking/contracts/10020328/timeline
01/22/2015 18:26:34|Trace|  Starting Web Driver
01/22/2015 18:26:35|Trace|  Making log in
01/22/2015 18:26:36|Trace|  Navigate to all user's contracts page
01/22/2015 18:26:36|Error|  No JSON on user's contracts page
01/22/2015 18:26:36|Info|  Making screenshot: ScreenShots\Contracts.jpg
01/22/2015 18:26:36|Trace|  Getting user's contracts JSON
01/22/2015 18:27:08|Trace|  Starting Web Driver
01/22/2015 18:27:13|Trace|  Making log in
01/22/2015 18:27:13|Trace|  Navigate to all user's contracts page
01/22/2015 18:27:14|Error|  No JSON on user's contracts page
01/22/2015 18:27:14|Info|  Making screenshot: ScreenShots\Contracts.jpg
01/22/2015 18:27:14|Trace|  Getting user's contracts JSON
01/22/2015 18:27:14|Trace|  Navigate to contract timeline event page 
01/22/2015 18:27:14|Error|  No JSON on contract timeline event page
01/22/2015 18:27:14|Info|  Making screenshot: ScreenShots\ContractTimelineEvent.jpg
01/22/2015 18:27:14|Trace|  Getting contract timeline event JSON
01/22/2015 18:27:45|Trace|  Starting Web Driver
01/22/2015 18:27:50|Trace|  Making log in
01/22/2015 18:27:52|Trace|  Navigate to all user's contracts page
01/22/2015 18:27:52|Error|  No JSON on user's contracts page
01/22/2015 18:27:52|Info|  Making screenshot: ScreenShots\Contracts.jpg
01/22/2015 18:27:52|Trace|  Getting user's contracts JSON
01/22/2015 18:30:53|Trace|  Starting Web Driver
01/22/2015 18:30:58|Trace|  Making log in
01/22/2015 18:31:00|Trace|  Navigate to all user's contracts page
01/22/2015 18:31:04|Trace|  Getting user's contracts JSON
01/22/2015 18:31:04|Info|  OK
01/22/2015 18:31:04|Trace|  Navigate to contract timeline event page 
01/22/2015 18:31:09|Trace|  Getting contract timeline event JSON
01/22/2015 18:31:09|Info|  OK
01/22/2015 18:34:15|Trace|  Starting Web Driver
01/22/2015 18:34:20|Trace|  Making log in
01/22/2015 18:34:22|Trace|  Navigate to credit contract
01/22/2015 18:34:28|Trace|  Getting credit contract JSON: contractNumber 10020328
01/22/2015 18:34:28|Info|  OK
01/22/2015 18:34:28|Trace|  Navigate to all user's contracts page
01/22/2015 18:34:28|Trace|  Getting user's contracts JSON
01/22/2015 18:34:28|Info|  OK
01/22/2015 18:34:28|Trace|  Navigate to contract timeline event page 
01/22/2015 18:34:29|Trace|  Getting contract timeline event JSON
01/22/2015 18:34:29|Info|  OK
01/22/2015 18:35:33|Trace|  Starting Web Driver
01/22/2015 18:35:38|Trace|  Making log in
01/22/2015 18:35:39|Trace|  Navigate to contract services 
01/22/2015 18:35:39|Trace|  Getting JSON /banking/contracts/10020328/services
01/22/2015 18:35:39|Info|  OK
01/23/2015 18:06:45|Trace|  Navigate to authentication page
01/23/2015 18:06:45|Trace|  Filling authentication fields
01/23/2015 18:06:46|Trace|  Click Registration Button 
01/23/2015 18:06:46|Trace|  Agree with license terms
01/23/2015 18:06:46|Trace|  Choose registration with account
01/23/2015 18:06:46|Trace|  Enter Account Number and access code: 40817810200240259336 123456
01/23/2015 18:06:46|Trace|  Click Next. Go to SMS code page
01/23/2015 18:07:18|Trace|  Enter SMS code(otp)
01/23/2015 18:07:18|Trace|  Click Next. Go to username page
01/23/2015 18:07:18|Trace|  Enter username: u5008556
01/23/2015 18:07:18|Trace|  Delete existing username field
01/23/2015 18:07:18|Trace|  Click Next. Go to password page
